hs/src/Reach/EditorInfo.hs

```diff
@@ -9,10 +9,19 @@ import Reach.AST.Base
 import Reach.AST.SL
 import qualified Reach.Eval.Core as C
 
+customConfig :: A.Config
+customConfig = A.Config {
+  confIndent = A.Spaces 4,
+  confCompare = compare,
+  confNumFormat = A.Generic,
+  confTrailingNewline = False
```

Comment:
For what it's worth I prefer a trailing newline.

Reply:
I did try it with `confTrailingNewline = True`, but that gives us an additional blank line, followed by a newline.

Reply:
That said, don't let my bikeshedding about newlines hold you up.

Reply:
If we do `confTrailingNewline = True`, we get

```
...
  "wait": {
    "CompletionItemKind": "Function"
  },
  "while": {
    "CompletionItemKind": "Keyword"
  },
  "with": {
    "CompletionItemKind": "Keyword"
  },
  "yield": {
    "CompletionItemKind": "Keyword"
  }
}

hamir@WSL2:~/reach-lang/hs$ 
```

Reply:
With `confTrailingNewline = False`, we get

```
  "wait": {
    "CompletionItemKind": "Function"
  },
  "while": {
    "CompletionItemKind": "Keyword"
  },
  "with": {
    "CompletionItemKind": "Keyword"
  },
  "yield": {
    "CompletionItemKind": "Keyword"
  }
}
hamir@WSL2:~/reach-lang/hs$ 
```

Reply:
Oh, yeah, because it's printed with `putStrLn`.  That's what I get for commenting without looking at the context.

Reply:
Ha, no worries.

It might be more intuitive if we do a `putStr` + `confTrailingNewline = True` combo, instead of `putStrLn` + `confTrailingNewline = False`.

Reply:
#648 hopefully makes this code more clear.